example/android/gradlew

```diff
@@ -1,25 +1,43 @@
-#!/usr/bin/env bash
+#!/usr/bin/env sh
```

Comment:
Could you explain why we should prefer `sh` over `bash`?
I'm used to go for `bash` as default.

Reply:
Please note that this file is automatically created by the Gradle `wrapper` task. As to why the Gradle devs decided to go for `sh`, I guess that's for better portability across platforms.